webapp/packages/plugin-connection-search/src/Search/ConnectionSearchService.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observable, makeObservable } from 'mobx';
+
+import { AdministrationScreenService } from '@cloudbeaver/core-administration';
+import { ConnectionInfoResource } from '@cloudbeaver/core-connections';
+import { injectable } from '@cloudbeaver/core-di';
+import { NotificationService } from '@cloudbeaver/core-events';
+import type { AdminConnectionSearchInfo } from '@cloudbeaver/core-sdk';
+import { PublicConnectionFormService } from '@cloudbeaver/plugin-connections';
+
+@injectable()
+export class ConnectionSearchService {
+  hosts = 'localhost';
+  databases: AdminConnectionSearchInfo[];
+
+  disabled = false;
+
+  // get disabled(): boolean {
+  //   return this.createConnectionService.disabled;
+  // }
+
+  // set disabled(value: boolean) {
+  //   this.createConnectionService.disabled = value;
+  // }
+
+  constructor(
+    private readonly notificationService: NotificationService,
+    private readonly connectionInfoResource: ConnectionInfoResource,
+    // private createConnectionService: CreateConnectionService,
+    private readonly administrationScreenService: AdministrationScreenService,
+    private readonly publicConnectionFormService: PublicConnectionFormService
+  ) {
+    makeObservable(this, {
+      hosts: observable,
+      databases: observable,
+      disabled: observable,
+    });
+
+    this.databases = [];
+    this.search = this.search.bind(this);
+    this.change = this.change.bind(this);
+    this.select = this.select.bind(this);
+  }
+
+  close(): void {
+    this.hosts = 'localhost';
+    this.databases = [];
+  }
+
+  async load(): Promise<void> {
+    if (this.administrationScreenService.isConfigurationMode && this.databases.length === 0) {
+      await this.search();
+    }
```

Comment:
configuration mode can be safely removed (we can't create connections in easy-config, or use public interface)